creative-works/sample-tweets-PORTUGUESE.txt

```diff
@@ -0,0 +1,13 @@
+Tweet padrão:
+
+Por favor RT:
+Estágios do Outreachy são 100% remotos, com bolsa de $5,500, e duram três meses.
```

Comment:
Oh, those tweets were fun to review. Here's my two cents (we'll be using all 280 characters in this one):

```suggestion
Estágios do Outreachy são 100% remotos, com bolsa de U$5.500, e duram 3 meses.
```